.travis.yml

```diff
@@ -7,16 +7,15 @@ cache:
     - $HOME/.composer/cache/files
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
   - 5.6
   - hhvm
+  - 7.0
+  - 7.1
 
 matrix:
   include:
-    - php: 5.5
-      env: SYMFONY_VERSION='2.3.*'
+    - php: 5.6
+      env: SYMFONY_VERSION='2.8.*'
     # Test against dev dependencies
     - php: 5.6
```

Comment:
I suggest moving this job to 7.1 (latest dev version of Symfony is not compatible with 5.6 for instance)